### PR TITLE
Added sending email

### PR DESCRIPTION
**Pull request by kujawad**

### CODE REVIEW

File: src/main/resources/application-local.properties [hunk 1]
```diff
@@ -4,8 +4,11 @@
...
 spring.datasource.password=postgres
-
 # ===============================
 # = JPA
 # ===============================
 spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
+# ===============================
+# = STATIC
+# ===============================
+local.url=dev-meta-bug.herokuapp.com/ticket/
```
Comment (krzjez) [inline src/main/resources/application-local.properties]: Is it correct local url?

File: src/main/java/com/metabug/service/EmailService.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    @Autowired
+    public EmailService(final JavaMailSender javaMailSender,
+                        final TemplateEngine templateEngine) {
+        this.javaMailSender = javaMailSender;
+        this.templateEngine = templateEngine;
+    }
+
+    public void send(final long ticketId, final String ticketTitle,
+                     final String ticketStatus, final String developer,
+                     final String to) {
```
Comment (krzjez) [inline src/main/java/com/metabug/service/EmailService.java]: "to" -> "recipient"

File: src/main/java/com/metabug/spring/MailConfiguration.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    private String username;
+
+    @Value("${spring.mail.password}")
+    private String password;
+
+    @Bean
+    public JavaMailSender javaMailSender() {
+        final JavaMailSenderImpl mailSender = new JavaMailSenderImpl();
+        mailSender.setHost("smtp.gmail.com");
+        mailSender.setPort(587);
```
Comment (krzjez) [inline src/main/java/com/metabug/spring/MailConfiguration.java]: You set those in application.properties, so maybe you can use them here, like you use username and password?

**Review by krzjez — APPROVED** [review]

File: src/main/resources/application-local.properties (hunk 1 above)
Comment (kujawad) [reply src/main/resources/application-local.properties]: fixed

File: src/main/java/com/metabug/service/EmailService.java (hunk 2 above)
Comment (kujawad) [reply src/main/java/com/metabug/service/EmailService.java]: done

File: src/main/java/com/metabug/spring/MailConfiguration.java (hunk 3 above)
Comment (kujawad) [reply src/main/java/com/metabug/spring/MailConfiguration.java]: removed configuration class

**Pull request merged**